app/Nova/User.php

```diff
@@ -196,6 +197,17 @@ public function fields(Request $request): array
                         ->help('This flag is set by JEDI but may be out of sync with ClickUp in some cases.'
                             .' It only controls UX elements.'),
 
+                    Text::make('Autodesk', 'autodesk_email')
+                    ->hideFromIndex()
+                    ->rules('nullable', 'max:255', 'email')
+                    ->creationRules('unique:users,autodesk_email')
+                    ->updateRules('unique:users,autodesk_email,{{resourceId}}'),
+
+                    Boolean::make('Autodesk Invite Pending', 'autodesk_invite_pending')
+                    ->hideFromIndex()
+                    ->help('This flag is set by JEDI but may be out of sync with Autodesk in some cases.'
+                           .' It only controls UX elements.'),
+
```

Comment:
Checks didn't catch it but the indentation is off here. The content of the lines are fine.